packages/mobile/src/select/Select.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {FunctionComponent, ReactNode, Fragment} from 'react'
+
+import {MenuItem} from '../menu'
+import {DropUp} from '../drop-up'
+
+import {SelectInput, SelectControl, Box, Pos, Card, MenuControl, styled, InputField, Icon, Spacer, OptionModel} from '@qiwi/pijma-core'
+
+const CardPos = Card.withComponent(Pos)
```

Comment:
https://www.styled-components.com/docs/api:
"As of styled-components v4 the withComponent API is now a candidate for deprecation. In all likelihood, you probably want to use the new "as" prop to simply switch what element/component being rendered since the withComponent API is destructive toward styles if the lowest-wrapped component is a StyledComponent."